### PR TITLE
Add Lightweight Run Management System

### PR DESCRIPTION
**Pull request by ddasilva**
This merge request adds a lightweight run management system, and updates the README to provide installation instructions. I also updated `setup_minimum_required_folders_files.py` to download the files to a run directory instead of $suncet_data. Long-term it probably makes sense to ship the metadata definitions with the repository and not have it in the run directory.

Explanation of the run management system (copied from Slack)
```
How you would run the code:
Step 1: Clone the repo
Step 2: python make_run.py --run-name NAME_OF_RUN                   # this makes a directory processing_runs/NAME_OF_RUN and puts an editable config.ini in there
Step 3: cp TELEMEATARY_PATH/*.bin procesing_runs/NAME_OF_RUN/input  # add input files
Step 4: python suncet_processing.py --run-name NAME_OF_RUN          # begins processing the run and writes to sunset_data=processing_runs/NAME_OF_RUN/level{1,2,3}_output 
Step 5: cp processing_runs/NAME_OF_RUN/level3/*  /ftp/public/       # copy output data to export directory

If you want to delete the run:
   rm -r processing_runs/NAME_OF_RUN
```

Some other observations:
* I think we can remove `setup.py` and `version`, because the code isn't meant to be installed as a package
* The tests need some love-- i can patch this up next

image

image

image

### CODE REVIEW

File: README.md
Comment (starfleetjames) [inline README.md:1]: Typo: `TELEMEATARY_PATH` should be `TELEMETRY_PATH`

**Comment by starfleetjames** [comment]
This looks great! Ready to merge on all aspects save one point to discuss: 

the `suncet_data` environment variable is intended to point to a local install of [this folder ](https://drive.google.com/drive/folders/1VrOe9QO6zBLfA_N7xqLTB3OMTnjeEi6N?usp=sharing) for any of the machines that are doing the routine processing. For other users that just are setting up the repo to start with minimal stuff, that's why the setup minimum shell script exists. So I think you're right to make this code change. Given that bit of background, do you anticipate any issues?

**Comment by ddasilva** [comment]
No, I think we're good to keep `$suncet_data` pointing to that data directory! I'd recommend `suncet_processor.py` also accept something like `--suncet-data-dir=` for people who don't want to muck around with environmental variables

File: README.md
Comment (ddasilva) [reply README.md:1]: Fixed!

**Pull request merged**